p2p/messageMonitor.go

```diff
@@ -135,10 +135,7 @@ func (ms *MessageService) startLoop() {
 				}
 			default:
 				ms.netService.node.logger.Error("Received unknown message.")
-				time.Sleep(5 * time.Millisecond)
```

Comment:
better to keep this sleep, when a large number of forged (unknown) messages are received, it maybe cost high cpu loading

Reply:
all msgs from messageCh is using subscribe by (ms *MessageService) Start, unknown msgs can not send to messageCh.